tests/PHPStan/Rules/Exceptions/CaughtExceptionExistenceRuleTest.php

```diff
@@ -0,0 +1,29 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Rules\Exceptions;
+
+class CaughtExceptionExistenceRuleTest extends \PHPStan\Rules\AbstractRuleTest
+{
+
+	protected function getRule(): \PHPStan\Rules\Rule
```

Comment:
@ondrejmirtes I recommend to add docblocks at every method that has parameters or returns something. this will help the IDE for a better integration.